### PR TITLE
fixes issue 475 MSSQL Unqualified Schema Table Parsing

### PR DESCRIPTION
**Pull request by T-Santos**
Resolves https://github.com/macbre/sql-metadata/issues/475

### CODE REVIEW

File: test/test_mssql_server.py [hunk 1]
```diff
@@ -1,6 +1,37 @@
...
+        ),
+        pytest.param(
+            "SELECT * FROM [my_server].[mydb].[dbo].[test_table]",
+            ["[my_server].[mydb].[dbo].[test_table]"],
+            id="With linked-server and object identifier delimiters",
+        ),
+    ],
+)
+def test_simple_queries_tables(query, expected):
+    assert Parser(query).tables == expected
```
Comment (macbre) [inline test/test_mssql_server.py:32]: Thanks for adding the test coverage!

**Comment by macbre** [comment]
@T-Santos - thanks for the fix, merging.

**Comment by T-Santos** [comment]
@macbre if you have a moment, would you be able to put out a python release so I'm able to pull the package (or inform me how I'm able to do so. I poked around a bit and didn't see any options for me to trigger a release but I certainly could have missed it). Thanks!

**Comment by macbre** [comment]
Done in https://github.com/macbre/sql-metadata/releases/tag/v2.14.0

**Pull request merged**